weblate/templates/dashboard/user.html

```diff
@@ -1,10 +1,22 @@
 {% extends "base.html" %}
 {% load i18n %}
+{% load static %}
 {% load permissions %}
 {% load translations %}
+{% load compress %}
 {% load crispy_forms_tags %}
 {% load icons %}
 
+{% block extra_script %}
+  {% compress js %}
+    <script defer data-cfasync="false" src="{% static 'vendor/moment.js' %}{{ cache_param }}"></script>
+    <script defer data-cfasync="false" src="{% static 'vendor/daterangepicker.js' %}{{ cache_param }}"></script>
+  {% endcompress %}
+  {% compress css %}
+    <link rel="stylesheet" href="{% static 'vendor/daterangepicker.css' %}{{ cache_param }}" />
+  {% endcompress %}
+{% endblock %}
```

Comment:
This needs to be loaded on several other pages where reports are available (currently component and project, category is requested as well in https://github.com/WeblateOrg/weblate/issues/11739). I'd probably include it in the main template instead.

Reply:
I noticed that the reports form is used by other pages also. I have included it to the other pages using the `extra_script` block.